config/environments/development.rb

```diff
@@ -30,6 +30,8 @@
   # Store uploaded files on the local file system (see config/storage.yml for options)
   config.active_storage.service = :local
 
+  config.action_mailer.delivery_method = :smtp
```

Comment:
Actually you don't to send emails on development right?

Take a look and add this [gem](https://github.com/ryanb/letter_opener). Letter opener allows you to simulate the email sending in development environment.

Reply:
No, actually I don't use it. Great! I'll use it in next features.